ftl/node/main.py

```diff
@@ -47,9 +47,16 @@
     help='The path where the application data sits.')
 
 
-def main():
+def main(base="", name="", directory=""):
```

Comment:
Ah sorry, I wasn't very clear here. Could you pass sys.argv to main from the bottom of this file, and pass that parameter to parser.parse_args?